functions/.eslintrc.json

```diff
@@ -82,7 +82,7 @@
 		"max-len": ["error", {
 			"code": 100
 		}],
-		"max-lines-per-function": ["error", 25],
+		"max-lines-per-function": ["error", 20],
```

Comment:
Are we going with 20 or 30, there seems to be a conflict between your choices?

Reply:
I was checking out something and forgot to revert back to 20. So i just did now. Its 20 from the original spec given to us.

Reply:
OK cause I saw 30 in the previous PR and also in Faith's change